docs/ocis/flow-docs/login-flow.md

```diff
@@ -26,35 +26,35 @@ sequenceDiagram
     participant ldap as External User Directory
 
     user->>+client: What is the content of my home?
-        client->>+proxy: PROPFIND <br> no (or expired) auth
-        Note over client,proxy: ocis needs to know the IdP that is<br>used to authenticate users. The<br>proxy will redirect unauthenticated<br>requests to that IdP.
+        client->>+proxy: PROPFIND\n no (or expired) auth
```

Comment:
This is the only line that replaces `<br>` with `\n`. Also other lines containing similar syntax don't do this. Intentional?